tests/unit_tests/dagmc/test_lost_particles.py

```diff
@@ -62,20 +62,20 @@ def broken_dagmc_model(request):
     return model
 
 
-def test_lost_particles(run_in_tmpdir, broken_dagmc_model):
+def test_lost_particles(broken_dagmc_model):
```

Comment:
Why was `run_in_tmpdir` removed here?

Reply:
Wanted the `model.xml` when I was testing locally and forgot to re-add it. My bad!